spec.html

```diff
@@ -37175,12 +37175,11 @@ <h1>
             1. If _rer_.[[IgnoreCase]] is *false*, return _ch_.
             1. Assert: _ch_ is a UTF-16 code unit.
             1. Let _cp_ be the code point whose numeric value is the numeric value of _ch_.
-            1. Let _u_ be toUppercase(« _cp_ »), according to the Unicode Default Case Conversion algorithm.
-            1. Let _uStr_ be CodePointsToString(_u_).
-            1. If the length of _uStr_ ≠ 1, return _ch_.
-            1. Let _cu_ be _uStr_'s single code unit element.
-            1. If the numeric value of _ch_ ≥ 128 and the numeric value of _cu_ &lt; 128, return _ch_.
-            1. Return _cu_.
+            1. Let _mapped_ be toUppercase(« _cp_ »), according to the Unicode Default Case Conversion algorithm.
+            1. If the length of _mapped_ ≠ 1 or the length of CodePointsToString(_mapped_) ≠ 1, return _ch_.
```

Comment:
```suggestion
            1. If the number of elements in _mapped_ ≠ 1 or the length of CodePointsToString(_mapped_) ≠ 1, return _ch_.
```

Strings are measured with length, Lists are measured with the number of elements.

Reply:
(There are, I think, 10 places in the spec that talk about the 'length' of a List.)

Reply:
Updated.